src/ProjectSystem/Environment.fs

```diff
@@ -149,21 +149,21 @@ module Environment =
 
   /// because 3.x is the minimum SDK that we support for FSI, we want to float to the latest
   /// 3.x sdk that the user has installed, to prevent hard-coding.
-  let latest3xSdkVersion sdkRoot =
+  let latest3xSdkVersion dotnetRoot =
```

Comment:
renamed these parameters to reflect their true purpose. SDK root in the current lingo means `dotnet-root/sdk`, but what was passed in to these functions is the `dotnet-root`.